### ISSUE TITLE
Kanban Board list

### ISSUE THREAD

**Issue by BDavydov** [3drepo/3drepo.io#1938]
- [x] [Comment tooltip is blocked by the column name](https://github.com/3drepo/3drepo.io/issues/1938#issuecomment-600591186)
- [x] [Issue names do not display tooltip on hover](https://github.com/3drepo/3drepo.io/issues/1938#issuecomment-600592156)
- [x] [Creating a new issue starts with a comment input & allows Untitled issues to be saved](https://github.com/3drepo/3drepo.io/issues/1938#issuecomment-600593309)
- [x] [Background is not consistent](https://github.com/3drepo/3drepo.io/issues/1938#issuecomment-601683334)
- [x] [New Issue window is not full height in Mozilla Firefox](https://github.com/3drepo/3drepo.io/issues/1938#issuecomment-606573009)
- [x] [Create new Issue should be greyed out until a Model/Federation has been selected](https://github.com/3drepo/3drepo.io/issues/1938#issuecomment-606660406)

**Comment by BDavydov** [comment]
### Description
When you hover over the comment field, you get a tooltip. Unfortunately, the comment tooltip is covered by another field giving information about the column category. 

### Steps to replicate
- Open Kanban board
- Load any model/federation that contains issues.
- Hover over the comments line of any issue.

### Pictures/ Gifs of the error
![Comment Hover](https://user-images.githubusercontent.com/19904513/76959939-787ce400-6912-11ea-9d4d-65030cb6a40b.jpg)

**Comment by BDavydov** [comment]
### Description
When you are hovering over the issue name, it does not display the tooltip. Issues with long names can not be read.

### Steps to replicate
- Open Kanban board
- Load any model/federation that contains issues.
- Hover over the Issue Name line

### Pictures/ Gifs of the error
![Name hover](https://user-images.githubusercontent.com/19904513/76960437-5df73a80-6913-11ea-803a-ef23533c3a20.gif)

**Comment by BDavydov** [comment]
### Description
When you are hovering over the issue name, it does not display the tooltip. Issues with long names can not be read.

### Steps to replicate
- Open Kanban board
- Load any model/federation that contains issues.
- Click on the + button to create a new issue
- The new issue starts with a comment input field instead of Issue Title
- The untitled issue can be saved 'Untitled Issue'

### Pictures/ Gifs of the error
![Create a new Issue](https://user-images.githubusercontent.com/19904513/76960528-90089c80-6913-11ea-8706-5e168b8cf6c5.gif)

**Comment by BDavydov** [comment]
### Description
The background is not consistent. Should be white on the left all the way down.

### Steps to replicate
- Open Kanban board
- Load any model/federation that contains issues.
- Click on the + button to create a new issue
- Start commenting on the issues
- After a while, the issue comments list grows to push the issue info up
- Grey space floats below the issue information

![image](https://user-images.githubusercontent.com/19904513/77164820-f2df6c80-6aa8-11ea-9670-db5bc13a5783.png)

**Comment by BDavydov** [comment]
### Description
When User creates a new issue using Mozilla Firefox, not the entire card is shown. User needs to scroll down the card to click submit an issue. Please make the card set at full height as per Google Chrome.

### Steps to replicate
- Open Mozilla Firefox
- Log in to 3D Repo
- Open Issues & Risks board
- Choose any project and click on the Green Plus button
- Issue card pops up, but the save button is not there

![Mozilla Kanban new issue](https://user-images.githubusercontent.com/19904513/78022045-327d4280-734c-11ea-90c5-aeb2d65be04e.gif)

**Comment by carmenfan** [comment]
**Create new Issue should be greyed out until a Model/Federation has been selected**

At the moment you can click on the button to add new issue prior to a model has been selected. This seems to bug out the whole feature completely if you try. It should be disabled until a model has been selected.
